netprofile_tickets/netprofile_tickets/views.py

```diff
@@ -817,6 +821,36 @@ def _send_ticket_mail(req, ticket=None, change=None):
                 continue
             recipient_map[addr] = tplvars
 
+    if ticket.assigned_user and user_setting(
```

Comment:
Might be a better idea to add temp subscription objects instead of using this block of code. See ```_default_subscriptions()```.